clap-v3-utils/src/input_parsers/mod.rs

```diff
@@ -22,8 +22,7 @@ pub mod signer;
     note = "Please use the functions in `solana_clap_v3_utils::input_parsers::signer` directly instead"
 )]
 pub use signer::{
-    pubkey_of_signer, pubkeys_of_multiple_signers, pubkeys_sigs_of, resolve_signer, signer_of,
-    STDOUT_OUTFILE_TOKEN,
+    pubkey_of_signer, pubkeys_of_multiple_signers, resolve_signer, signer_of, STDOUT_OUTFILE_TOKEN,
```

Comment:
This is public interface, so I think we should keep the re-export for now.
Technically we can break things in v2, but it probably makes sense to remove these things all at once then.

Reply:
Oh oops, I did not realize that I removed this re-export.

Reply:
Ah it is because rust gives a warning that a deprecated function is re-exported. I re-added the re-export and added `allow(deprecated)`.